src/main/java/ch/ethz/idsc/sophus/app/filter/NonuniformGeodesicCausalFilterDemo.java

```diff
@@ -26,6 +26,7 @@ public NonuniformGeodesicCausalFilterDemo() {
   protected void updateStateTime() {
     super.updateStateTime();
     Scalar interval = RationalScalar.of(spinnerRadius.getValue(), 10);
+    // TODO OB is Se2Geodesic.INSTANCE sufficiently generic here?
```

Comment:
Yes I would say that Se2 is generic for this specific demo since we define stateTime as (state = xya, time = t). However, we should generalize this demo for other geodesics